doc/api/child_process.md

```diff
@@ -998,6 +1010,10 @@ The `'message'` event is triggered when a child process uses
 The message goes through serialization and parsing. The resulting
 message might not be the same as what is originally sent.
 
+If the `advancedSerialization` option was used when spawning the child process,
```

Comment:
Is this `advancedSerialization` option mentioned in the docs? If not, maybe something like
```md
If the `serialization` option was set to `'advanced'` when spawning the child
process...
```